cycode/cli/printers/table_printer.py

```diff
@@ -1,170 +1,168 @@
+            rows.append(detection.detection_details.get('commit_id', ''))
 
-        click.echo(text_table.draw())
+        return rows
 
-    @staticmethod
-    def set_table_width(headers: List[str], text_table: Texttable) -> None:
-        header_width_size_cols = []
-        for header in headers:
-            header_len = len(header)
-            if header == CVE_COLUMN:
-                header_width_size_cols.append(header_len * 5)
-            elif header == UPGRADE_COLUMN:
-                header_width_size_cols.append(header_len * 2)
-            else:
-                header_width_size_cols.append(header_len)
-        text_table.set_cols_width(header_width_size_cols)
-
-    @staticmethod
-    def _print_summary_issues(detections: List, title: str) -> None:
-        click.echo(f'⛔ Found {len(detections)} issues of type: {click.style(title, bold=True)}')
-
-    def _get_common_detection_fields(self, detection: Detection) -> List[str]:
-        row = [
-            detection.detection_details.get('file_name'),
-            detection.detection_details.get('ecosystem'),
-            detection.detection_details.get('package_name'),
-            detection.detection_details.get('is_direct_dependency_str'),
-            detection.detection_details.get('is_dev_dependency_str')
-        ]
+    def _get_detection_code_segment_fields(self, detection: Detection, document: Document) -> List[str]:
+        detection_details = detection.detection_details
 
-        if self._is_git_repository():
-            row = [detection.detection_details.get('repository_name')] + row
+        detection_line = detection_details.get('line_in_file', -1)
+        if self.scan_type == SECRET_SCAN_TYPE:
+            detection_line = detection_details.get('line', -1)
 
-        return row
+        detection_position = get_position_in_line(document.content, detection_details.get('start_position', -1))
+        violation_length = detection_details.get('length', -1)
 
-    def _is_git_repository(self) -> bool:
-        return self.context.obj.get("remote_url") is not None
-
-    def _get_upgrade_package_vulnerability(self, detection: Detection) -> List[str]:
-        alert = detection.detection_details.get('alert')
-        row = [
-            detection.detection_details.get('advisory_severity'),
-            *self._get_common_detection_fields(detection),
-            detection.detection_details.get('vulnerability_id')
+        rows = [
+            detection_line,
+            detection_position,
         ]
 
-        upgrade = ''
-        if alert.get("first_patched_version"):
-            upgrade = f'{alert.get("vulnerable_requirements")} -> {alert.get("first_patched_version")}'
-        row.append(upgrade)
+        if self.scan_type == SECRET_SCAN_TYPE:
+            rows.append(f'{violation_length} chars')
+
+            violation = ''
+
+            file_content_lines = document.content.splitlines()
+            if detection_line < len(file_content_lines):
```

Comment:
from the results you attach in slack it seems you print only the line of the secret?

Reply:
you are right. tbh i have no clue how to make it pretty. i mean how to show the full line or few lines of the code in the small cell of the table. for example for sast scan results. that's why i just show only hardcoded secret for now